apps/hash-ai-worker-ts/src/activities/infer-entities/persist-entities/update-entities.ts

```diff
@@ -111,8 +112,12 @@ export const updateEntities = async ({
               ...properties,
             };
 
+            const draft = !!extractDraftIdFromEntityId(
+              existingEntity.metadata.recordId.entityId,
+            );
```

Comment:
Not to solve now, we'll need to think about how the 'find existing entity to update' logic works once we have multiple drafts of an entity, and a live one... whether the AI is proposing updates to the live entity, or to one of the drafts edits to it – and if the latter, how we choose between them (multiple people might each have a draft working copy of the same entity)

cc @nonparibus

Reply:
We want each attribute of an entity to be independently governed, so we can diff two entities, and only overwrite/update changed or different attribute values, in the case of multiple drafts (i.e. basically supporting "merge" of two entities).